custom/icds_reports/utils/__init__.py

```diff
@@ -477,7 +477,8 @@ def zip_folder(pdf_files):
     return zip_hash
 
 
-def create_pdf_file(pdf_hash, pdf_context):
+def create_pdf_file(pdf_context):
+    pdf_hash = uuid.uuid4().hex
```

Comment:
nit: this isn't technically a hash—maybe call it pdf_uuid?

Reply:
Agreed. In the follow up ticket I opened to start using BlobExpiration, I  also mentioned that instead of using uuid, we should hash it on the config. Leaving that until then